src/main/java/com/purbon/kafka/topology/Constants.java

```diff
@@ -82,7 +82,9 @@
   public static final String GROUP_MANAGED_PREFIXES = "topology.group.managed.prefixes";
 
   public static final String PLATFORM_SERVERS_CONNECT = "platform.servers.connect";
-  public static final String PLATFORM_SERVER_KSQL = "platform.server.ksql";
+  public static final String PLATFORM_SERVER_KSQL_URL = "platform.server.ksql";
+  public static final String PLATFORM_SERVER_KSQL_BASIC_AUTH_USER = "platform.server.ksql-user";
```

Comment:
what about using "platform.server.ksql.user" ? instead of separating with a dash.

Reply:
I think in that case platform.server.ksql needs to be renamed to platform.server.ksql.url as well, otherwise the typesafe config mapping library complains that platform.server.ksql is not a string but an object. It would become a backward incompatible change. I'd also prefer it separated with . instead of -

Reply:
yap, I agree and no problem with making it backwards incompatible, only think about:

* This PR will go into the 3.x release.
* We should note the change as backwards-incompatible in the docs and release notes.

I would love to release 3.x  soon so I kinda prefer doing it good.

what do you think?